src/components/LikesLabel.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+function LikesLabel({ likes }) {
+  return <div className="ui olive big circular label"> {likes} </div>;
```

Comment:
If this component only has this <div>, it's not worth having a seperate component for it. Add that div line directly where you need it.